index.html

```diff
@@ -408,6 +408,10 @@ <h4>Farhat Naik</h4>
             title: "You don’t have to change the world. You just have to change what you pay attention to in the world. And that, it turns out, is hugely powerful",
             author: "Vishen Lakhiani"
         },
+        {
+            title: "A person might be an expert in any field of knowledge or a master of many material skills and accomplishments. But without inner cleanliness his brain is a desert waste.",
+            author: "Anmol Yadav"
```

Comment:
Put the name of original author and add minimum 3 quotes

Reply:
@VighneshManjrekar i have made the requested changes, kindly check my PR